charts/dapr/charts/dapr_scheduler/values.yaml

```diff
@@ -27,6 +27,9 @@ ports:
   etcdGRPCPeerPort: 2380
 
 replicaCount: 1
+ha:
+  # etcd should have an odd number of replicas https://etcd.io/docs/v3.3/faq/ to guarantee quorum
+  replicaCount: 3
```

Comment:
yes, was just trying to separate it for HA but without relying on global.ha one.  Let me know how you'd rather have it, it will be an easy change. we could just have a hardcoded one on global.ha.enabled=true (ie 3) and use local chart one if bigger that it.

Reply:
Taking the `ha` value if enabled, with a larger `replicaCount` number that overrides makes sense to me.